src/kixi/heimdall/group.clj

```diff
@@ -0,0 +1,15 @@
+(ns kixi.heimdall.group
+  (:require [kixi.heimdall.components.database :as db]
+            [kixi.heimdall.util :as util]
+            [qbits.alia.uuid :as uuid]))
+
+(defn create!
+  [session {:keys [owner_id] :as group}]
+  (let [group-id (uuid/random)
+        group-data (assoc group
+                          :id group-id
+                          :created (util/db-now)
```

Comment:
groups should also have an optional name field, so we can have designated groups (GLA, etc etc)